### PR TITLE
Add filtering security scans by ip

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by ar4s — APPROVED** [review]

File: src/ralph/security/api.py [hunk 1]
```diff
@@ -71,7 +72,9 @@ def to_internal_value(self, data):
         if errors:
             raise serializers.ValidationError(errors)
         data['base_object'] = base_object.pk
-        result = super(SaveSecurityScanSerializer, self).to_internal_value(data)
+        result = super(
```
Comment (szok) [inline src/ralph/security/api.py]: result = super().to_internal_value(data)

Python 3 :)

**Review by szok — APPROVED** [review]

File: src/ralph/security/api.py [hunk 2]
```diff
@@ -80,6 +83,19 @@ class SecurityScanViewSet(RalphAPIViewSet):
     serializer_class = SecurityScanSerializer
     save_serializer_class = SaveSecurityScanSerializer
 
+    def _filter_by_ip(self, queryset):
```
Comment (mkurek) [inline src/ralph/security/api.py]: use `FilterSet`, similar to https://github.com/allegro/ralph/blob/ng/src/ralph/assets/api/filters.py#L4
Using this, this field will appear in OPTIONS result

**Review by mkurek — APPROVED** [review]

File: src/ralph/security/api.py (hunk 2 above)
Comment (ghost) [reply src/ralph/security/api.py]: done

File: src/ralph/security/api.py [hunk 3]
```diff
@@ -71,15 +73,26 @@ def to_internal_value(self, data):
...
+        name='base_object__ethernet_set__ipaddress__address'
+    )
+
+
 class SecurityScanViewSet(RalphAPIViewSet):
     queryset = SecurityScan.objects.all()
     serializer_class = SecurityScanSerializer
     save_serializer_class = SaveSecurityScanSerializer
 
+    def get_queryset(self):
```
Comment (mkurek) [inline src/ralph/security/api.py]: no need for this imo

**Review by mkurek — CHANGES_REQUESTED** [review]

File: src/ralph/security/api.py (hunk 3 above)
Comment (ghost) [reply src/ralph/security/api.py]: done

**Review by mkurek — APPROVED** [review]

**Pull request merged**